bodhi/server/renderers.py

```diff
@@ -98,11 +100,21 @@ def link_dict(obj):
                 return dict(href=request.route_url(route, **{param: obj[key]}))
             return link_dict
 
+        def describe_update(notes, builds):
+            def describe(*args, **kwargs):
+                text = '# Packages in this update:\n'
+                for p in builds(*args, **kwargs):
+                    text += '* ' + p.nvr + '\n'
+                text += '# Update description:\n' + notes(*args, **kwargs)
+                return markup(None, text)
+            return describe
+
         getters = {
             'updates': {
                 'title': operator.itemgetter('alias'),
-                'link': linker('update', 'id', 'title'),
-                'description': operator.itemgetter('notes'),
+                'link': linker('update', 'id', 'alias'),
```

Comment:
This change was also made in https://github.com/fedora-infra/bodhi/pull/3239

Reply:
I've removed the commit from this PR